### DISCUSSION TITLE
Specifying Database Schema at Initilization

### DISCUSSION THREAD

**Discussion by siraphobk** [ory/kratos#1280] [Q&A] [answered]
I am currently working on an application that already contains many tables. I want to integrate Kratos into my current database rather than create a separate database for it because I need to link identity to my current tables. 

The problem is I want to separate my domain tables and Kratos tables using PostgreSQL schema. Now, everything is in the default schema; `public`. I know I could create my domain tables in a different schema in the first place. However, I would want to know if there's a way to specify the target schema for Kratos's tables? Can it be done by setting the DSN?

**Comment by aeneasr** [comment, marked as answer]
Sorry for the late reply! We use pgx under the hood ( https://github.com/jackc/pgx ) which I believe allows setting `?search_path=foo,bar,public` in the DSN, so e.g. `postgres://user:password@host:123/dbname?search_path=...`

**Reply by siraphobk** [reply]
Thank you for your reply. I'll try it out. :100:

**Reply by iglesiasroberto14** [reply]
Did it work?
